examples/composite_template/example_application_window/mod.rs

```diff
@@ -0,0 +1,26 @@
+mod imp;
+
+use glib::subclass::prelude::*;
+use gtk::prelude::*;
+use gtk::{gio, glib};
+
+glib::wrapper! {
+    pub struct ExampleApplicationWindow(ObjectSubclass<imp::ExampleApplicationWindow>)
+        @extends gtk::Widget, gtk::Window, gtk::ApplicationWindow, @implements gio::ActionMap, gio::ActionGroup;
+}
+
+impl ExampleApplicationWindow {
+    pub fn new<P: glib::IsA<gtk::Application>>(app: &P) -> Self {
+        glib::Object::new(&[("application", app)])
+            .expect("Failed to create ExampleApplicationWindow")
+    }
+
+    pub fn init_label(&self) {
```

Comment:
This doesn't have to be public as it's only used by the `imp` submodule. Should never really be called from the outside